docs/content/docs/connectors/table/hive/hive_dialect.md

```diff
@@ -346,33 +344,33 @@ CREATE FUNCTION function_name AS class_name;
 DROP FUNCTION [IF EXISTS] function_name;
 ```
 
-## DML
-
-### INSERT
+## DML & DQL _`Beta`_
 
-```sql
-INSERT (INTO|OVERWRITE) [TABLE] table_name [PARTITION partition_spec] SELECT ...;
-```
+Hive dialect supports a commonly-used subset of Hive's [DML](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DML)
+and [DQL](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Select). The following lists some examples of
+HiveQL supported by the Hive dialect.
 
-The `partition_spec`, if present, can be either a full spec or partial spec. If the `partition_spec` is a partial
-spec, the dynamic partition column names can be omitted.
+- [SORT/CLUSTER/DISTRIBUTE BY](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+SortBy)
+- [Grouping Sets and GROUPING__ID function](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+GroupBy#LanguageManualGroupBy-GroupingSets,Cubes,Rollups,andtheGROUPING__IDFunction)
+- [INSERT INTO dest schema](https://issues.apache.org/jira/browse/HIVE-9481)
+- [LATERAL VIEW](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+LateralView)
+- [Implicit type conversions](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-AllowedImplicitConversions)
```

Comment:
Are all the supported syntax are listed above? e.g. Select, Join, Union, Subqueries, Lateral views, Over, CTE. 

https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Select#LanguageManualSelect-MoreSelectSyntax

Reply:
Good point. I have added these since they're widely used. But it's difficult to make an exhaustive list of supported features.